src/System.Runtime/tests/System/TimeSpanTests.netcoreapp.cs

```diff
@@ -22,6 +22,27 @@ private static IEnumerable<object[]> MultiplicationTestData()
             yield return new object[] {TimeSpan.MaxValue, 0.5, TimeSpan.FromTicks((long)(long.MaxValue * 0.5))};
         }
 
+        private static IEnumerable<object[]> ParseSpecialCasesData()
```

Comment:
Is there any more descriptive naming we could use about what makes these special?

Reply:
I changed the name and added a comment to tell more what we are trying to do. thanks.